File: frontend/src/app/services/models/reservation-summary-employee-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ClientNameDto } from '../models/client-name-dto';
import { FlightBasicDto } from '../models/flight-basic-dto';
export interface ReservationSummaryEmployeeDto {
  clientDto?: ClientNameDto;
  flightDto?: FlightBasicDto;
  id?: string;
  reservationDate?: string;
  status?: 'PENDING' | 'ACTIVE' | 'CANCELLED' | 'EXPIRED' | 'COMPLETED' | 'FAILED';
}

File: src/main/java/FlyAway/reservation/dto/ReservationSummaryEmployeeDto.java
package FlyAway.reservation.dto;

import FlyAway.client.dto.ClientNameDto;
import FlyAway.flight.dto.FlightBasicDto;
import FlyAway.reservation.ReservationStatus;

import java.time.LocalDateTime;
import java.util.UUID;

public record ReservationSummaryEmployeeDto(
        UUID id,
        ReservationStatus status,
        LocalDateTime reservationDate,
        ClientNameDto clientDto,
        FlightBasicDto flightDto

) {
}
